backend/classes/Game.js

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
folder name rather be something that has to do with out project(maybe models, or something like that)

Reply:
floder name changed